en/StopTimeEvent.md

```diff
@@ -0,0 +1,7 @@
+### StopTimeEvent
+
+| Field Name | Recommendation |
+| --- | --- |
+| delay | If only `delay` is provided in a `stop_time_update` `arrival` or `departure` (and not `time`), then the GTFS [stop_times.txt](https://gtfs.org/reference/static#stopstxt) must contain arrival_times and/or departure_times for these corresponding stops. A `delay` value in the real-time feed is meaningless unless you have a clock time to add it to in the GTFS `stop_times.txt` file. |

```

Comment:
One of six places where we use "real-time". Though that hyphenated presentation is the way it lives out in the universe a lot of the time, do we want to use it in the document when we're talking about "GTFS-realtime"?